CRITICAL - 2023-08-16 02:13:32 --> Undefined variable $rol_usuario
in APPPATH\Controllers\Cotizacion.php on line 140.
 1 APPPATH\Controllers\Cotizacion.php(140): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $rol_usuario', 'C:\\xampp\\htdocs\\proyectosistema\\app\\Controllers\\Cotizacion.php', 140)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cotizacion->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizacion))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 02:13:51 --> Undefined variable $rol_usuario
in APPPATH\Controllers\Cotizacion.php on line 140.
 1 APPPATH\Controllers\Cotizacion.php(140): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $rol_usuario', 'C:\\xampp\\htdocs\\proyectosistema\\app\\Controllers\\Cotizacion.php', 140)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cotizacion->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizacion))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 02:44:37 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`dscriptn_db_sistema`.`personas`, CONSTRAINT `fk_personas_cargos1` FOREIGN KEY (`id_cargo`) REFERENCES `cargos` (`id`)) in C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\proyectosistema\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\proyectosistema\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectosistema\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectosistema\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectosistema\app\Controllers\Persona.php(379): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(934): App\Controllers\Persona->guardar()
#9 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#10 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectosistema\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-16 02:44:37 --> Cannot add or update a child row: a foreign key constraint fails (`dscriptn_db_sistema`.`personas`, CONSTRAINT `fk_personas_cargos1` FOREIGN KEY (`id_cargo`) REFERENCES `cargos` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'0911127520\', \'1\', \'1234567890\', \'asdas@asdas.com\', 0, \'\', 1, 0, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'0911127520\', \'1\', \'1234567890\', \'asdas@asdas.com\', 0, \'\', 1, 0, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'0911127520\', \'1\', \'1234567890\', \'asdas@asdas.com\', 0, \'\', 1, 0, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (:nombres:, :identificacion:, :es_extranjero:, :telefono:, :correo:, :es_empleado:, :id_cargo:, :es_cliente:, :es_proveedor:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Persona.php(379): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 02:44:56 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`dscriptn_db_sistema`.`personas`, CONSTRAINT `fk_personas_cargos1` FOREIGN KEY (`id_cargo`) REFERENCES `cargos` (`id`)) in C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\proyectosistema\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\proyectosistema\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectosistema\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectosistema\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectosistema\app\Controllers\Persona.php(379): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(934): App\Controllers\Persona->guardar()
#9 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#10 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectosistema\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-16 02:44:56 --> Cannot add or update a child row: a foreign key constraint fails (`dscriptn_db_sistema`.`personas`, CONSTRAINT `fk_personas_cargos1` FOREIGN KEY (`id_cargo`) REFERENCES `cargos` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'45645645\', \'1\', \'12345679\', \'johndavis@gmail.com\', 0, \'\', 1, 0, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'45645645\', \'1\', \'12345679\', \'johndavis@gmail.com\', 0, \'\', 1, 0, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (\'persona test\', \'45645645\', \'1\', \'12345679\', \'johndavis@gmail.com\', 0, \'\', 1, 0, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `personas` (`nombres`, `identificacion`, `es_extranjero`, `telefono`, `correo`, `es_empleado`, `id_cargo`, `es_cliente`, `es_proveedor`, `estado`) VALUES (:nombres:, :identificacion:, :es_extranjero:, :telefono:, :correo:, :es_empleado:, :id_cargo:, :es_cliente:, :es_proveedor:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Persona.php(379): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
